content-management-system/src/main/java/com/cms/example/cms/entities/enums/Gender.java

```diff
@@ -0,0 +1,8 @@
+package com.cms.example.cms.entities.enums;
+
+// Gender Enum
```

Comment:
Is this comment carrying any meaning?  If not remove it

Reply:
removed